web-client/src/presenter/computeds/messageDocumentHelper.js

```diff
@@ -52,6 +54,9 @@ export const messageDocumentHelper = (get, applicationContext) => {
     caseDocument &&
     caseDocument.eventCode === INITIAL_DOCUMENT_TYPES.petition.eventCode;
 
+  const isStipulatedDecision =
+    caseDocument && caseDocument.eventCode === STIPULATED_DECISION_EVENT_CODE;
```

Comment:
Seems like if we have an empty `caseDocument` object, we should be doing something about that sooner, which would allow the rest of this function to be more concise.

Reply:
Good point, I think that was a bit of unnecessary defensive coding.

Reply:
Updated!

Reply:
thank you!